### ISSUE TITLE
Fix Astropy intersphinx linking

### ISSUE THREAD

**Issue by pllim** [nmearl/pyfocal#70]
I have `'sphinx.ext.intersphinx'` as part of Sphinx extensions in `conf.py`. I pointed intersphinx mapping for Astropy to ` 'astropy': ('http://docs.astropy.org/en/latest/', None)` and yet my attempt to link to Astropy modeling API doc (e.g., `~astropy.modeling.models.BrokenPowerLaw1D`) is unsuccessful.

@eteq , do you happen to know why?

**Comment by pllim** [comment]
Turns out I need to set this in `conf.py`
```python
default_role = 'py:obj'
```
**and**
explicitly refer to the actual module the class resides in, like
```python
`~astropy.modeling.powerlaws.BrokenPowerLaw1D`
```

**Comment by pllim** [comment]
Fixed in #77
